packages/manager/src/features/VPC/VPCLanding/VPCLanding.tsx

```diff
@@ -133,6 +159,12 @@ const VPCLanding = () => {
         page={pagination.page}
         pageSize={pagination.pageSize}
       />
+      <VPCDeleteDialog
+        id={deleteDialog.id}
+        label={deleteDialog.label}
+        onClose={closeDeleteDialog}
+        open={deleteDialog.open}
+      />
```

Comment:
We could spread `deleteDialog` props here.
```suggestion
      <VPCDeleteDialog
       {...deleteDialog}
        onClose={closeDeleteDialog}
      />
```